auth0/src/test/java/com/auth0/android/authentication/AuthenticationAPIClientTest.kt

```diff
@@ -1624,8 +1712,8 @@ public class AuthenticationAPIClientTest {
         )
     }
 
-    private fun bodyFromRequest(request: RecordedRequest): Map<String, String> {
-        val mapType = object : TypeToken<Map<String?, String?>?>() {}.type
+    private fun <T> bodyFromRequest(request: RecordedRequest): Map<String, T> {
```

Comment:
this change required me to specify the type in all of the tests that were using it above